### PR TITLE
Automate comment of every PR merge. #1033

### PR DESCRIPTION
**Pull request by Anshika14528**
## Description

As soon as a PR merge, a bot will add this message
Screenshot 2024-05-24 at 9 11 41 PM

Great work @{username} !!
Do not forget to
+ Share your contribution on socials and tag Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [x] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots

> Uploading a screenshot illustrating the approach you developed for validation is required.

## Additional Notes/Comments

> Any additional remarks or suggestions concerning this pull request are welcome.

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

**Comment by Anshika14528** [comment]
@nikhil25803 sir, can you review this?
Sorry for being late it took me 3 days to figure this out

**Comment by Anshika14528** [comment]
@nikhil25803  sir , Can you review this?
and also make sure that you have these function on
Screenshot 2024-05-27 at 3 03 30 PM

**Comment by Anshika14528** [comment]
@nikhil25803 sir, please review it and merge it.

**Comment by nikhil25803** [comment]
Hey @Anshika14528, test this out it one of your repo and upload a screenshot/video here to validate if it is working.

File: .github/workflows/pr_merged.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Auto Comment on PR Merged
+
+on:
+  pull_request_target:
+    types: [closed]
```
Comment (nikhil25803) [inline .github/workflows/pr_merged.yml:5]: The type should not be `closed` ideally, I want the message to be sent only if it is merged successfully.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Comment by Anshika14528** [comment]
Screenshot 2024-06-06 at 10 01 51 PM
It is checking here that if it is merged or not

**Review by nikhil25803 — APPROVED** [review]

**Pull request merged**